### ISSUE TITLE
[A11y]Screen reader starts over from beginning when applying filters.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9422]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1749979

Environment Details:
App name: NuGet Service
#URL: [https://www.nuget.org](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=05%7c01%7cv-nitipi%40microsoft.com%7c9a84307e7aa744a4b24208dafa5ca2fe%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638097573052590837%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=C8Eab3i6ixAmqsOjG4AxNT36z9WZQiXPbjDDkMcS5ME%3D&reserved=0)

OS and Browser Details:
Edition Windows 11 Enterprise
OS build 22621.1105.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)
Screen reader: JAWS, NVDA

Repro Steps:
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
"Packages returned for json" screen gets opened.
Tab till filter present at left side of screen and apply any filter.
Observe that screen reader is announcing all the details from the beginning when we apply filter.

Actual Result:
Screen reader is announcing all the details from the beginning when we apply filter.

Expected Result:
Screen reader should not announce all the details from the beginning when we apply filter, it should only announce the status message that "812 packages returned for json"

Notes:
Unfortunately, I believe this is relatively nontrivial to fix on our side. From my understanding of this issue and the feature, this is caused by the fact that hitting applying appear to redo the search with filters enabled, causing a page refresh. When the page refreshes, context on keyboard nav etc is lost. In theory, it should be possible to pass context to the next page, but I am unsure that this the right fix long term.
